controllers/app/kogitobuild_controller_test.go

```diff
@@ -74,8 +74,8 @@ func TestReconcileKogitoBuildSimple(t *testing.T) {
 	sort.SliceStable(kogitoISList.Items, func(i, j int) bool {
 		return kogitoISList.Items[i].Name < kogitoISList.Items[j].Name
 	})
-	assert.Equal(t, kogitobuild.GetDefaultBuilderImage(), kogitoISList.Items[0].Name)
-	assert.Equal(t, kogitobuild.GetDefaultRuntimeJVMImage(), kogitoISList.Items[1].Name)
+	assert.Equal(t, kogitobuild.GetDefaultBuilderImage(), kogitoISList.Items[1].Name)
+	assert.Equal(t, kogitobuild.GetDefaultRuntimeJVMImage(), kogitoISList.Items[0].Name)
```

Comment:
I didn't spot this, I am curios as well.

Reply:
@spolti @radtriste 
Are changed because the new names are in different alphabetically orde previously the name was

- kogito-builder
- kogito-runtime-jvm

now are 

- kogito-runtime-jvm
- kogito-s2-builder

and are returned in this way by 
kubernetes.ResourceC(cli).ListWithNamespace(t.Name(), kogitoISList)